### ISSUE TITLE
Bug: did:webs at root of website not being resolved correctly

### ISSUE THREAD

**Issue by wip-abramson** [peacekeeper/did-webs-iiw-tutorial#5]
Not sure if this is a bug in the demo, or a bug in keri.

But after seeing this demo at IIW I had a go at creating my own did:webs.

The first one I created was:
  did:webs:drwip.com:EI-m1H12TiwNTfOL9ccGnS_dsSSZBlLf-MUW08SsUDk0

See files
- https://drwip.com/EI-m1H12TiwNTfOL9ccGnS_dsSSZBlLf-MUW08SsUDk0/did.json
- https://drwip.com/EI-m1H12TiwNTfOL9ccGnS_dsSSZBlLf-MUW08SsUDk0/keri.cesr

Then I attempted to resolve it in the uniresolver: [https://dev.uniresolver.io/#did:webs:drwip.com:EI-m1H12TiwNTfOL9ccGnS_dsSSZBlLf-MUW08SsUDk0](https://dev.uniresolver.io/#did:webs:drwip.com:EI-m1H12TiwNTfOL9ccGnS_dsSSZBlLf-MUW08SsUDk0)

This gave me weird results. There were no errors returned, but also no did document and no verificationMethod.

So, I attempted to resolve it locally using the kli from the docker container.

`dkr did webs resolve --name controller --did did:webs:drwip.com:EI-m1H12TiwNTfOL9ccGnS_dsSSZBlLf-MUW08SsUDk0`

This gave me an error:

`requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://drwip.com/None/EI-m1H12TiwNTfOL9ccGnS_dsSSZBlLf-MUW08SsUDk0/did.json`

Note: the URL contains /None/

So I tried another approach. Where the DIDs files were not at the root of the site but under a folder.

did:webs:drwip.com:test:ECBqw4j6nLmwvNVWQkAqHcak-u3NdAqHHMXeJf1falqS

- https://drwip.com/test/ECBqw4j6nLmwvNVWQkAqHcak-u3NdAqHHMXeJf1falqS/did.json
- https://drwip.com/test/ECBqw4j6nLmwvNVWQkAqHcak-u3NdAqHHMXeJf1falqS/keri.cesr

This worked: [https://dev.uniresolver.io/#did:webs:drwip.com:test:ECBqw4j6nLmwvNVWQkAqHcak-u3NdAqHHMXeJf1falqS](https://dev.uniresolver.io/#did:webs:drwip.com:test:ECBqw4j6nLmwvNVWQkAqHcak-u3NdAqHHMXeJf1falqS)

So basically there seems to be some bug in resolution, that adds None when there is no subfolder.

**Comment by peacekeeper** [comment]
Thanks for reporting this bug. I think this was fixed in https://github.com/hyperledger-labs/did-webs-resolver/commit/94e1f790b28df0d6b8665d05c2bf103720944e5f.

I will try to update the demo to use an updated Docker image, will let you know then.

**Comment by peacekeeper** [comment]
@wip-abramson I pushed new Docker images and updated the docker-compose.yml file to use the latest tag now:
https://github.com/peacekeeper/did-webs-iiw37-tutorial/commit/5365d2ff25a7c6fb8c624bc4d074c2810dfb8561

Could you maybe try again to resolve your `did:webs` using the latest image?

**Comment by wip-abramson** [comment]
Will do. Although this is also an issue with the uniresolver, want me to raise it over there as well?

**Comment by peacekeeper** [comment]
Not really necessary to also raise with Universal Resolver, we'll update the driver there "automatically" when we fix things here.

**Comment by wip-abramson** [comment]
Yep, this is fixed now. Thanks

**Comment by peacekeeper** [comment]
@wip-abramson Great, thanks for the confirmation. Can we close this?

**Comment by wip-abramson** [comment]
Yep
